components/bl-signature-pad-component/src/index.js

```diff
@@ -0,0 +1,54 @@
+import ActionButtons from './actionButtons.js';
+import SignaturePad from './leap/signature_pad.umd.min.js';
+
+export default function MyCustomComponent({ component, eventHandlers }) {
+  const elRef = React.useRef(null);
+  const signaturePad = React.useRef(null);
+  
+  function resizeCanvas() {
+    const ratio = Math.max(window.devicePixelRatio || 1, 1);
+    const canvas = elRef.current;
+    canvas.width = canvas.offsetWidth * ratio;
+    canvas.height = canvas.offsetHeight * ratio;
+    canvas.getContext("2d").scale(ratio, ratio);
+    signaturePad.current.clear();
+  }
+
+  React.useEffect(() => {
+    signaturePad.current = new SignaturePad(elRef.current, {
+      backgroundColor: 'rgb(255, 255, 255)',
+      penColor: component.penColor
+    });
+    
+    window.onresize = resizeCanvas;
+    resizeCanvas();
+  }, []);
+  
+  return (
+    <div className="bl-customComponent-signaturePad">
+      <div className="bl-customComponent-signaturePad-body">
```

Comment:
there is no need to have the root prefix `bl-customComponent-signaturePad` for each single element

```
//jsx

<div className="bl-customComponent-signature-pad">
      <div className="wrapper">
         <canvas 
```

```
//Less

.bl-customComponent-signature-pad {
   .wrapper {
      canvas {
      
       }
    }

 }
```